ehrql/loaders.py

```diff
@@ -47,6 +74,8 @@ def load_definition_in_subprocess(definition_type, definition_file, user_args, e
             "PYTHONPATH": os.environ.get("PYTHONPATH", ""),
             # Our entrypoint will emit warnings if we don't set this
             "PYTHONHASHSEED": "0",
+            # `pledge` requires this to be set
+            "TMPDIR": "/tmp",
```

Comment:
Is this always a reasonable value? Should we ask the OS?

Reply:
Quite right, I've amended this.